scripts/build-hyperdb.js

```diff
@@ -44,4 +74,10 @@ pearDB.collections.register({
   schema: '@pear/dht'
 })
 
+pearDB.collections.register({
+  name: 'bundle',
+  schema: '@pear/bundle',
+  key: ['link']
+})
```

Comment:
define an index over the tags also

add this

``` js
pearDB.require(path.join(__dirname, 'spec', 'helpers.js'))

pearDB.indexes.register({
  name: 'bundle-by-tags',
  collection: '@pear/bundle',
  unique: false,
  key: {
    type: {
      fields: [{
        name: 'tag',
        type: 'string'
      }]
    },
    map: 'tags'
  }
})
```

and inside the spec folder add a file, helpers.js that has

```
exports.tags = (record) => record.tags
```